File: MinimumMovestoEqualArrayElements.cpp
#include<iostream>
#include<vector>
#include<algorithm>

using namespace std;

class Solution {
public:
    int minMoves(vector<int>& nums) {
        
        sort(nums.begin(), nums.end());
        
        int count = 0;
        for(int i = nums.size() - 1; i > 0; i--) {
        	count += nums[i] - nums[0];
		}
		
		return count;
        
    }
};

int main() {
	
	int n;
	cin>>n;
	
	vector<int> nums;
	for(int i = 0; i < n; i++) {
		int num;
		cin>>num;
		nums.push_back(num);
	}
	
	Solution *solution = new Solution();
	cout<<solution->minMoves(nums);
	
	return 0;
	
}

File: MinimumMovestoEqualArrayElements.py
class Solution:
    def minMoves(self, nums) -> int:
        return sum(nums) - len(nums) * min(nums)


if __name__ == '__main__':
    nums = [int(num) for num in input().split()]
    solution = Solution()
    print(solution.minMoves(nums))
